content/content/hackathon_2020_savethedate.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "OHBM hackathon 2020 - Save the date"
+date: 2020-01-26T21:55:26+01:00
+draft: false
+sidebar: true # or false to display the sidebar
+sidebarlogo: fresh-white-alt # From (static/images/logo/)
+include_footer: true
+---
+
+![](landing_hackathon_2020.jpg)
+
+
+
+## What is a hackathon?
```

Comment:
Also it seems that the mark up of the different tiltes and some other markdown features is ignored.